tests/playwright/sanity/container.test.js

```diff
@@ -43,6 +43,205 @@ test.describe( 'Container tests', () => {
+		// Flex-direction: column
+		await page.click( '.elementor-control-flex_direction i.eicon-arrow-down' );
+		// Align items: flex-start
+		await page.click( '.elementor-control-flex_align_items i.eicon-align-start-v' );
+		// Set `min-height` to test if there are `flex-grow` issues.
+		await page.locator( '.elementor-control-min_height .elementor-control-input-wrapper input' ).fill( '1500' );
+
+		// Assert
+		expect( await container.screenshot( {
+			type: 'jpeg',
+			quality: 70,
+		} ) ).toMatchSnapshot( 'container-column-full-start.jpeg' );
+
+		// Act
+		await wpAdmin.selectElement(  editor, containerId );
+		// Content Width: boxed
+		await page.selectOption( '.elementor-control-content_width >> select', 'boxed' );
+
+		// Assert
+		expect( await container.screenshot( {
+			type: 'jpeg',
+			quality: 70,
+		} ) ).toMatchSnapshot( 'container-column-boxed-start.jpeg' );
+
+		await wpAdmin.setExperiments( {
+			container: false,
+		} );
+	} );
+
+	test( 'Test widgets inside the container using position absolute', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: true,
+		} );
+
+		const editor = await wpAdmin.useElementorCleanPost();
+
+		// Close Navigator
+		await editor.closeNavigator();
+
+		// Set Canvas template.
+		await editor.pageLayoutTemplate( 'canvas' );
+
+		const container = await editor.addElement( { elType: 'container' }, 'document' ),
+			pageView = editor.getPreviewFrame().locator( 'body' );
+
+		// Act.
+		// Add widget.
+		await editor.addWidget( 'heading', container );
+		// Select container.
+		await editor.selectElement( container );
+		// Set position absolute.
+		await wpAdmin.activatePanel( 'advanced' );
+		await page.waitForSelector( '.elementor-control-position >> select' );
+		await page.selectOption( '.elementor-control-position >> select', 'absolute' );
+		await page.locator( '.elementor-control-z_index .elementor-control-input-wrapper input' ).fill( '50' );
+		await page.locator( '.elementor-control-_offset_x .elementor-control-input-wrapper input' ).fill( '50' );
+		await page.locator( '.elementor-control-_offset_y .elementor-control-input-wrapper input' ).fill( '50' );
+
+		// Assert
+		// Take screenshot.
+		expect( await pageView.screenshot( {
+			type: 'jpeg',
+			quality: 70,
+		} ) ).toMatchSnapshot( 'heading-boxed-absolute.jpeg' );
+
+		// Act
+		// Select container.
+		await editor.selectElement( container );
+		// Set full content width
+		await wpAdmin.activatePanel( 'layout' );
+		await page.selectOption( '.elementor-control-content_width >> select', 'full' );
+
+		// Assert
+		expect( await pageView.screenshot( {
+			type: 'jpeg',
+			quality: 70,
+		} ) ).toMatchSnapshot( 'heading-full-absolute.jpeg' );
+
+		await wpAdmin.setExperiments( {
+			container: false,
+		} );
+	} );
+
+	test( 'Test widgets inside the container using position fixed', async ( { page }, testInfo ) => {
+		// Arrange.
```

Comment:
I split the test for postion absolute and position fixed into two tests. It was difficult to finetuning one long test.